micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/svgindex.js

```
@@ -677,6 +677,13 @@ const ReceiptIcon = () => (
   </svg>
 );
 
+const ReceiptInboxIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 22" width="24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)